inlong-audit/audit-sdk/src/main/java/org/apache/inlong/audit/send/SenderChannel.java

```diff
@@ -0,0 +1,103 @@
+/**
```

Comment:
The implementation of this part is relatively small, can you write one yourself?